packages/workers-shared/router-worker/src/index.ts

```diff
@@ -129,7 +129,7 @@ export default {
 			if (userWorkerInvocation) {
 				// Don't send user Worker errors to sentry; we have no way to distinguish between
 				// CF errors and errors from the user's code.
-				return;
+				throw err;
```

Comment:
If you are now throwing, is the comment above still accurate?

Reply:
Returning `undefined` in a `fetch()` will throw an error anyway, so I haven't changed the behavior here: just "fixed" the implicit return type to `Promise<Response>` rather than `Promise<Response | undefined>` which it was before (needed as part of the [TS update commit](https://github.com/cloudflare/workers-sdk/pull/8279/commits/f57d1f830e9f29472d4af77a6491b605e5eba890).

It still doesn't log to Sentry since that would happen below only if we didn't return or throw here: https://github.com/cloudflare/workers-sdk/pull/8279/files#diff-d666c170036315c34fb63daeaa6abe85b7f96a3facd431c49ae1d7b8777f0869R138-R140